### PR TITLE
add: alert component

### PR DESCRIPTION
**Pull request by Shubhamkr2610**
build alert component with side bar

### CODE REVIEW

File: components/alert/alert.css [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    margin: 50px 0px 0px 0px;
+    display: flex;
+}
+
+.alert-content{
+    margin-left: 5%;
+}
+.alert{
+    text-align: center;
+    padding: 0px 20px 0px 20px;
```
Comment (itsmebipul00) [inline components/alert/alert.css]: use rems

File: components/alert/alert.css [hunk 2]
```diff
@@ -0,0 +1,54 @@
+@import url("/index.css");
```
Comment (varun13169) [inline components/alert/alert.css:1]: For local files following will work.
@import "/index.css";

Not sure if using URL will have a performance impact or not, so please read about it.

File: components/alert/alert.css [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+@import url("sidebar.css");
+
+*{
+    margin:0;
+    padding: 0;
```
Comment (varun13169) [inline components/alert/alert.css]: 0 bananas ??? or 0 lightyear ???

mention units.

File: components/alert/alert.css [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+}
+body{
+    background-image: none;
+}
+
+.alert-sidebar-main-container{
+    margin: 50px 0px 0px 0px;
+    display: flex;
+}
```
Comment (varun13169) [inline components/alert/alert.css:15]: use rem, please.

50px = 50/16 rem ~ 3.12rem

File: components/alert/sidebar.css [hunk 5]
```diff
@@ -0,0 +1,40 @@
+*{
```
Comment (developer-anand) [inline components/alert/sidebar.css]: you can put it in your index css so you don't have to write it again and again.

File: index.css [hunk 6]
```diff
@@ -31,6 +41,8 @@ body {
     color: white;
     margin-left: 1100px;
     margin-top: -60px;
+    
+
```
Comment (varun13169) [inline index.css:45]: why this is part of reset CSS ???

**Review by varun13169 — CHANGES_REQUESTED** [review]

File: components/alert/alert.css (hunk 3 above)
Comment (Shubhamkr2610) [reply components/alert/alert.css]: What should I use px or rem unit?

File: components/alert/sidebar.css (hunk 5 above)
Comment (Shubhamkr2610) [reply components/alert/sidebar.css:1]: Okay . I'll read about it and update in what will correct way to import file.

File: components/alert/alert.css (hunk 2 above)
Comment (Shubhamkr2610) [reply components/alert/alert.css:1]: Okay . I'll read about it and update in what will correct way to import file.

File: components/alert/alert.css (hunk 1 above)
Comment (Shubhamkr2610) [reply components/alert/alert.css]: I have  made the changes . Please check

File: components/alert/sidebar.css (hunk 5 above)
Comment (Shubhamkr2610) [reply components/alert/sidebar.css]: Moved it  to index.css Please look into it

File: components/alert/alert.css (hunk 3 above)
Comment (Shubhamkr2610) [reply components/alert/alert.css]: I have made the changes using px. Can you give your feedback . Is it right or not to use px?

File: components/alert/alert.css (hunk 4 above)
Comment (Shubhamkr2610) [reply components/alert/alert.css:15]: Changed to rem. Please review it

File: components/alert/sidebar.css [hunk 7]
```diff
@@ -0,0 +1,33 @@
+.aside-left{
+    width: 30%;
+    height: 100vh;
+   
+    /* padding: 20px; */
+    /* background-color: rgb(221, 151, 21);   */
+}
+
+h3{
+    font-size: 35px;
```
Comment (humzahasan) [inline components/alert/sidebar.css]: Can this be refactored to rem ?

**Review by humzahasan — APPROVED** [review]
Few of the mesurments are in px, refacor to rem to maintain consistency

File: components/alert/sidebar.css (hunk 7 above)
Comment (Shubhamkr2610) [reply components/alert/sidebar.css]: yes.. of-course

Comment (Shubhamkr2610) [reply components/alert/sidebar.css]: made the changes . please review

**Review by Muzammil-khan710 — COMMENTED** [review]
Remove comment code from index.css file..

File: components/alert/alert.html [hunk 8]
```diff
@@ -0,0 +1,130 @@
...
+
+    <!---------------------------------main context end------------------->
+
+
+    <!--------------------------------footer starts------------------------>
+    <footer class="footer">
+
+        <h2>Made with ❤️ by shubham</p>
+            <h3>connect with me 👇 </p>
+                <a href="https://github.com/Shubhamkr2610">github<i class="fab fa-github-square"></i></a>
```
Comment (arjun-computer-geek) [inline components/alert/alert.html]: Always use target=_blank with the anchor attributes for external links

File: components/alert/sidebar.css [hunk 9]
```diff
@@ -0,0 +1,32 @@
+.aside-left{
+    width: 30%;
+    height: 100vh;
+    /* padding: 20px; */
```
Comment (arjun-computer-geek) [inline components/alert/sidebar.css]: Always remove unused code

**Review by arjun-computer-geek — CHANGES_REQUESTED** [review]
Do some changes

File: components/alert/sidebar.css (hunk 7 above)
Comment (Shubhamkr2610) [reply components/alert/sidebar.css]: removed the commented and unused code. please review .

File: components/alert/sidebar.css (hunk 9 above)
Comment (Shubhamkr2610) [reply components/alert/sidebar.css]: removed . please review it.

**Comment by Shubhamkr2610** [comment]
> 



> Remove comment code from index.css file..

okay, removed . please review it.

File: components/alert/alert.html (hunk 8 above)
Comment (Shubhamkr2610) [reply components/alert/alert.html]: I implemented this to my code. please check and review it

**Review by developer-anand — APPROVED** [review]
Good to go.

**Review by arjun-computer-geek — APPROVED** [review]
Now looks good to me

**Pull request merged**